### ISSUE TITLE
does this data layers support multi currency ?

### ISSUE THREAD

**Issue by adventuretocode** [HassanAMZ/shopify-datalayer#1]
https://github.com/TechnicalWebAnalytics/dataLayer-shopify/issues/65

@HassanAMZ  if you can help. all the event are working this data layers ?

**Comment by HassanAMZ** [comment]
I don’t think I understood your question, can you rephrase it?

**Comment by adventuretocode** [comment]
@HassanAMZ  thank for the looking the issue.
The datalayers get the product price value based on the shop currency setup as of now . but my store has multiple currency added.
datalayers push shop settings currency with incorrect price for the all the event,

test case one store add  more then one currency option in store front. then check the which price and currency  is going. 

here data opt 
![image](https://user-images.githubusercontent.com/114073623/230306538-e6cdd9f7-8fcd-48f2-9783-f494573c57d6.png)

**Comment by HassanAMZ** [comment]
Thank you for the clarification.

All the multi currency apps have some sort of new variables in which the
store the currency data. You can read the relevant documentation of the app
and update the datalayer code’s liquid variable.

**Comment by adventuretocode** [comment]
@HassanAMZ   if you can share any reference that will really helpful.
also the main repo there are few event not working. that repo i think not actively  maintain.
here event list
 remove cart 
  add to cart dynamic
  checkout steps
  purchase event code not working 
  if you can improve in your repo that was really grateful.

**Comment by HassanAMZ** [comment]
Thanks, I'll check it out. However, I’m really swamped with a lot of other
projects. I might not even have time to look at it in near future.
I will keep it in mind and update whenever you get a chance.
